routes/api.php

```diff
@@ -93,9 +93,20 @@
 Route::post('/Sharing::add', [Administration\SharingController::class, 'add']);
 Route::post('/Sharing::delete', [Administration\SharingController::class, 'delete']);
 
-Route::post('/WebAuthn::register/gen', [Administration\WebAuthController::class, 'generateRegistration']);
-Route::post('/WebAuthn::register', [Administration\WebAuthController::class, 'verifyRegistration']);
-Route::post('/WebAuthn::login/gen', [Administration\WebAuthController::class, 'generateAuthentication']);
-Route::post('/WebAuthn::login', [Administration\WebAuthController::class, 'verifyAuthentication']);
-Route::post('/WebAuthn::list', [Administration\WebAuthController::class, 'list'])->middleware(['admin']);
-Route::post('/WebAuthn::delete', [Administration\WebAuthController::class, 'delete'])->middleware(['admin']);
+// Route::post('/WebAuthn::register/gen', [Administration\WebAuthController::class, 'generateRegistration']);
+// Route::post('/WebAuthn::register', [Administration\WebAuthController::class, 'verifyRegistration']);
+// Route::post('/WebAuthn::login/gen', [Administration\WebAuthController::class, 'generateAuthentication']);
+// Route::post('/WebAuthn::login', [Administration\WebAuthController::class, 'verifyAuthentication']);
```

Comment:
As always.

Reply:
Fixed in [4cdfc8c](https://github.com/LycheeOrg/Lychee/pull/1469/commits/4cdfc8cfffc94a3b571ec8c68b2fa6063a0e5d1f)